v_5_0_0/files/k8s-resource/calico-all.yaml

```diff
@@ -317,7 +333,7 @@ spec:
         # This container installs the CNI binaries
         # and CNI network config file on each node.
         - name: install-cni
-          image: {{ .RegistryDomain }}/giantswarm/cni:v3.9.1
+          image: {{ .RegistryDomain }}/giantswarm/cni:v3.10.1
```

Comment:
Shoud we ot use a helm variable for the image versions since we use the same one in multiple places?

Reply:
That is not super straight forward to do right now. We could add it through helm but then need to make sure it renders correctly always (like with the k8s version).

We can do it but I would rather work towards ignition operator where this gets easier.

Reply:
Thanks for the feedback :) I understand it is not our main area of focus

Reply:
the thing is that with calico it's not just bumping version, there are sometimes other changes to the manifest so someone just bumping the version might lead to the problem

We could put ti to some go variable as we do with k8s